src/app/core/utils/ComponentUtils.ts

```diff
@@ -108,7 +108,7 @@ export function CreateGroup(objects: IOObject[]): IOObjectSet {
  * @param  w The wire to start from
  * @return   The array of wires/WirePorts in this path (including w)
```

Comment:
Please add a short description of the `full` parameter to these javadoc comments